Python/bltinmodule.c

```diff
@@ -2484,7 +2484,7 @@ builtin_sum_impl(PyObject *module, PyObject *iterable, PyObject *start)
                 /* Single digits are common, fast, and cannot overflow on unpacking. */
                 switch (Py_SIZE(item)) {
                     case -1: b = -(sdigit) ((PyLongObject*)item)->ob_digit[0]; break;
-                    case  0: continue;
+                    case  0: Py_DECREF(item); continue;
```

Comment:
I'm thinking that the "continue" warrants a comment.  Inside a switch-case, a *break* ends a case but a *continue* means to go to the top of the loop.  The side-by-side use of *break* and *continue* is a bit confusing (I did a double take when I first saw it); hence, my suggestion to add a comment.

Reply:
:+1:

Reply:
Agreed. I actually didn't add it before to keep the PR contained. But makes sense. I have pushed a comment in the new commit